crypto/rijndael.c

```diff
@@ -1,5 +1,8 @@
 /****************************************************************************
  * crypto/rijndael.c
+ *
+ * SPDX-License-Identifier: ???
```

Comment:
'Public Domain'

Reply:
we ca not use PD

Reply:
The public domain is handled in a different way in EU and US